source/MRMesh/MRMeshTopology.h

```diff
@@ -66,6 +66,15 @@ class MeshTopology
     /// the cut in rings in both cases is made after a and b
     MRMESH_API void splice( EdgeId a, EdgeId b );
 
+    /// collapses given edge in a vertex and deletes
+    /// 1) faces: left( e ) and right( e );
+    /// 2) vertex org( e )/dest( e ) if given edge was their only edge, otherwise only dest( e );
+    /// 3) edges: e, next( e.sym() ), prev( e.sym() ), and optionally next( e ), prev( e ) if their left and right triangles are deleted;
```

Comment:
change edges and vertes? (if we delete optional edges from (3) then we should also delete optional org(e) from (2))?